vulnerabilities/importers/github.py

```diff
@@ -146,189 +122,339 @@
     "-class.-jw.util.version.Version-",
 }
 
+ECOSYSTEM_PKG_TYPE_MAPPING = {
+    "MAVEN": "maven",
+    "NUGET": "nuget",
+    "COMPOSER": "composer",
+    "PIP": "pypi",
+    "RUBYGEMS": "gem",
+    "GO": "golang",
+}
+
+ECOSYSTEM_VERSION_RANGE_MAPPING = {
+    "MAVEN": MavenVersionRange,
+    "NUGET": NugetVersionRange,
+    "COMPOSER": ComposerVersionRange,
+    "PIP": PypiVersionRange,
+    "RUBYGEMS": GemVersionRange,
+    "GO": GolangVersionRange,
+}
+
+PKG_TYPE_ECOSYSTEM_MAPPING = {value: key for (key, value) in ECOSYSTEM_PKG_TYPE_MAPPING.items()}
+
+QUERY = """
+        query{
+        securityVulnerabilities(first: 100, ecosystem: %s, %s) {
+            edges {
+            node {
+                advisory {
+                identifiers {
+                    type
+                    value
+                }
+                summary
+                references {
+                    url
+                }
+                severity
+                publishedAt
+                }
+                package {
+                name
+                }
+                vulnerableVersionRange
+            }
+            }
+            pageInfo {
+            hasNextPage
+            endCursor
+            }
+        }
+        }
+        """
+
 
 class GitHubTokenError(Exception):
     pass
 
 
+def get_item(json_object: dict, *attributes):
```

Comment:
This is not specifically JSON, correct? just some plain data dictionary that was loaded from JSON... but may be we do not need to have JSON in its name at this stage?